### ISSUE TITLE
Flutter plugin that configures notification overrides the configure done in application's native side. IOS.

### ISSUE THREAD

**Issue by cyberail** [MaikuB/flutter_local_notifications#2085]
Hello flutter team.

I am handling notifications from two SDKs , one of the SDKs namely Sendibrd_chat_sdk is written in dart and I am using flutter_local_notification package to handle notifications from, in parallel I have Sendbird_call_sdk written in swift configured as IOS native sdk and trying to handle the notifications from native swift, my swift code is written in /iOS folder of the application itself.

Problem is as soon as I have flutter_local_notification instantiated in flutter, IOS configuration of notifications written by me does not work anymore for the call_sdk it seems to be overridden by flutter_local_notification plugin.

I have 3-fold question:

I would like to confirm that it really is a known case for the IOS cause I seem to not be able to find any question or answer about this issue anywhere.(Part of the reason why I am writing it here)?

Is there any known solution for this ?

Would the behaviour persisted, if had created custom plugin and had moved my code from applications iOS folder into the custom plugins iOS folder ?

Thank you in advance.

**Comment by MaikuB** [comment]
What you're describing isn't about a known issue about this plugin per se. Rather, it's how the native iOS APIs only allows for one delegate where only there can be one handler for dealing with notification events. Supposedly there is a way to get around this via method sizzling and something you can look up. I'm not an iOS developer though so don't know how to implement this. This is why I put a request to flutter team to see if there's a different approach as they've done this for other parts of the ecosystem but it's been many years and possible the issue has been closed. If you can implement it then you can submit a PR back to this repository.

Other option is you look at writing your own code on the iOS side that does what you need specifically for your app

Note: I'm not part of the flutter nor is this plugin associated with them. This is a side project of mine that has had contributions from the community
